app/views/govuk_publishing_components/components/_notice.html.erb

```diff
@@ -11,10 +12,12 @@
   css_classes = %w(gem-c-notice)
   css_classes << (shared_helper.get_margin_bottom)
 
+  aria_attributes = aria_live ? {label: 'Notice', live: 'polite'} : {label: 'Notice'}
```

Comment:
should we allow the option for `assertive` too?

Reply:
I'm not sure if there's a use case for having `aria-live: assertive` on this component? I'm a bit wary of adding `assertive` if it's not really needed because it could have a negative effect on user experience if misused.

Reply:
I usually favour open APIs, hence allowing any value to `aria_live` instead of using a boolean flag. However, on govuk_publishing_components we have this 'principle' that we don't want to allow something unless it's necessary/there's a need for it - and this aligns with this principle.
We can see this in action on how the label for this component doesn't allow a custom value.

Maybe it's something to be discussed at one of our meetings.